### DISCUSSION TITLE
群晖启动失败

### DISCUSSION THREAD

**Discussion by crazyweeds** [omniedgeio/omniedge#43] [Q&A] [answered]
<img width="473" alt="image" src="https://user-images.githubusercontent.com/16688520/188149769-ca4922c5-b419-4ed1-b645-59d8d8532c0c.png">

无法启动，没有任何提示，日志在什么目录？

**Comment by yongqianme** [comment, marked as answer]
Answered in twitter: 

https://twitter.com/yongqianme/status/1565698253735313410?s=20&t=1g9paICNy9jzJWpSaUg6MQ
